extensions/email/src/main/java/org/mule/extension/email/internal/util/EmailConnectorUtils.java

```diff
@@ -48,6 +48,16 @@
     public static final String TEXT = "text/*";
 
     /**
+     * Default email content encoding
+     */
+    public static final String UTF_8 = "UTF-8";
```

Comment:
MediaType constants are not Strings, they are MediaType and cannot be used as a defaultValue in  the annotation. The alternative I can think of is having a constant in EmailConnectorUtils converting that MediaType.TEXT.toString() and using that as value in the annotation.